test-e2e/panels/filters.test.ts

```diff
@@ -57,7 +57,12 @@ async function baselineCorrectionFilter(nmrium: NmriumPage) {
     ),
   ).toBeVisible();
 }
-
+async function xScaleDomain(nmrium: NmriumPage, min: number, max: number) {
```

Comment:
Please add a method in NmriumPage called assertXScaleDomain(min, max).

We will often reuse it ! And don't hesiate to add new method in NmriumPage